src/main/java/racingcar/domain/RandomNumberGenerator.java

```diff
@@ -0,0 +1,18 @@
+package racingcar.domain;
+
+import camp.nextstep.edu.missionutils.Randoms;
+
+public class RandomNumberGenerator implements NumberGenerator {
+
+    private static final int START_INCLUSIVE = 0;
+
+    private final int endInclusive;
```

Comment:
왜 이건 상수로 만들지 않으셨나요?

Reply:
이것 또한 확장성을 고려하였는데 추후에 9가 아닌 10, 20과 같은 수도 가능하게 열어두기 위함입니다.

Reply:
네이밍 컨벤션을 통일해도 좋을 것 같습니다!
```suggestion
    private final int END_INCLUSIVE;
```